src/app/activity/bounty/Claim.ts

```diff
@@ -27,40 +24,27 @@ export const claimBounty = async (request: ClaimRequest): Promise<any> => {
         claimedBounty = await writeDbHandler(request, getDbResult.dbBountyResult, claimedByUser);
     }
     
-    let bountyEmbedMessage: Message;
-    // TODO: consider changing claim, submit, complete, and delete requests to have a channel id instead of the complete Message
-    if (!request.message) {
-        const bountyChannel: TextChannel = await claimedByUser.guild.channels.fetch(getDbResult.bountyChannel) as TextChannel;
-        bountyEmbedMessage = await bountyChannel.messages.fetch(getDbResult.dbBountyResult.discordMessageId).catch(e => {
-            LogUtils.logError(`could not find bounty ${request.bountyId} in discord #bounty-board channel ${bountyChannel.id} in guild ${request.guildId}`, e);
-            throw new RuntimeError(e);
-        });
-    } else {
-        bountyEmbedMessage = request.message;
-    }
-
-    // Need to refresh original bounty so the messages are correct
-    getDbResult = await getDbHandler(request); 
-    const createdByUser: GuildMember = await claimedByUser.guild.members.fetch(getDbResult.dbBountyResult.createdBy.discordId);
-    await claimBountyMessage(bountyEmbedMessage, claimedBounty, createdByUser, claimedByUser, getDbResult.dbBountyResult);
+    const claimedBountyCard = await BountyUtils.canonicalCard(claimedBounty._id, request.activity);
     
-    const bountyUrl = process.env.BOUNTY_BOARD_URL + claimedBounty._id;
-    const origBountyUrl = process.env.BOUNTY_BOARD_URL + getDbResult.dbBountyResult._id;
     let creatorClaimDM = 
```

Comment:
We should probably rename this to something more sensible like `creatorNotification`